ui/packages/catalog-ui-search/src/main/webapp/react-component/container/histogram/index.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import styled from '../../../react-component/styles/styled-components'
+
+const Empty = styled.div`
+  text-align: center;
+  padding: ${props => props.theme.largeSpacing};
+  display: none;
+`
+const Attribute = styled.div`
+  display: block;
+  opacity: 1;
+  transition: opacity ${props => props.theme.coreTransitionTime} linear;
+  transform: translateX(0%);
+`
+const NoData = styled.div`
+  text-align: center;
+  padding: ${props => props.theme.largeSpacing};
+  display: none;
+`
+const Container = styled.div`
+  display: block;
+  height: ~'calc(100% - 135px)';
+  opacity: 1;
+  transition: opacity ${props => props.theme.coreTransitionTime} linear;
+  transform: translateX(0%);
+`
+const Warning = styled.span`
+  color: ${props => props.theme.warningColor};
+`
+
+export const HistogramContainer = () => (
+  <React.Fragment>
+    <Empty className="histogram-empty">
```

Comment:
are the class names necessary? It looks like you deleted them in histogram.less

Reply:
It's still in there. The CSS toggles the display: none/block property. I couldn't replace that without migrating everything that CSS touches to React.